src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -158,9 +158,7 @@ class DocBaseViewer extends BaseViewer {
         super.setup();
 
         this.docEl = this.createViewer(document.createElement('div'));
-        this.docEl.setAttribute('aria-label', __('document_label'));
```

Comment:
We added this label to address a previous accessibility concern around [caret navigation](https://en.wikipedia.org/wiki/Caret_navigation). It seems unwise to remove it without a replacement solution.

Reply:
@jstoffan thanks for pointing this out
So the recommendations would be to change the aria label in this case to say "Document Viewer"
this is slightly redundant with the next element up in the DOM, which is a region called "File Preview." The main thing we need is to remove these confusing instructions

Reply:
I don't think that addresses the issue I mentioned with caret navigation. Can you sync up with the person requesting these changes to make sure we don't regress?

Reply:
in short, recommendation stays the same (maybe we should consider updating box keyboard docs). Added more detailed explanation from person requesting changes below.

_as a practice we should not have very specific, hidden help text in our UIs.n this case, our accessibility auditors called out that this text is inaccurate and counter productive. nothing about the current text tells a user how to successfully use caret browsing. caret browsing also differs between browsers and OSes, so if we were to have in-UI instructions, we would have very bloated instructions that can (and have) become obsolete as commands change over time.
it is important to note that caret browsing is generally useful for users with vision, as screen reader users can linearly navigate through a document without caret browsing, using default screen reader browse commands (although screen reader users might use caret browsing for text selection). users with vision are not helped by hidden text in the UI.
what we can do is update our [keyboard commands documentation](https://support.box.com/hc/en-us/articles/360044194873-Using-Keyboard-Shortcuts-for-Box) to include a section on how to use caret browsing with Preview. I don't think this needs to be a blocker for this bug fix though - the first order of business is to remove the confusing instructions_